src/plugins/maps_dashboards/public/index.scss

```diff
@@ -1 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+@import "maplibre-gl/dist/maplibre-gl.css";
+
 /* stylelint-disable no-empty-source */
```

Comment:
we can remove this.

Reply:
We need remain it for Maplibre right format  https://maplibre.org/maplibre-gl-js-docs/api/#maplibre-css.